src/ext/language_tools.js

```diff
@@ -14,8 +14,10 @@ var keyWordCompleter = {
         }
         var state = editor.session.getState(pos.row);
         var completions = session.$mode.getCompletions(state, session, pos, prefix);
+        completions = completions.map((el) => el.completerId = keyWordCompleter.id);
```

Comment:
do we have to set completerId for all completions or can do it only when needed?

Reply:
We could define it only when needed, but I suggest to leave this for default ones (to not interfere with user defined completers)